frontend/src/components/Start.js

```diff
@@ -0,0 +1,229 @@
+import styled from 'styled-components/macro'
+import startscreenJpg from './../assets/startscreen.jpg'
+import wandergoldSvg from './../assets/wandergold.svg'
+import closeSvg from './../assets/close.svg'
+import compassSvg from './../assets/compass.svg'
+import { useState, useEffect } from 'react'
+
+export default function Start() {
+  const [isSearchFocused, setSearchFocus] = useState(false)
+  const [coordsToSearch, setCoordsToSearch] = useState({
+    latitude: null,
+    longitude: null,
+    locationName: '',
+    googlePlaceId: '',
+  })
+  const [suggestionList, setSuggestionList] = useState([])
+
+  useEffect(() => console.log(coordsToSearch))
+
+  return (
+    <Wrapper>
+      <LogoArea onClick={() => setSearchFocus(false)}>
+        <h1>
+          <img src={wandergoldSvg} alt="wandergold" />
+        </h1>
+      </LogoArea>
+      <LocationSearch>
+        <SearchField className={isSearchFocused && 'active'}>
```

Comment:
It shouldn't be required to set a `className` if you have `styled-components`. 
Instead, you can style `SearchField` based on `isSearchFocused`.
See https://styled-components.com/docs/basics#styling-any-component

Example:

```suggestion
        <SearchField active={isSearchFocused} >
```

and a small change in your `SeachField` (I am not sure which properties you like to set -> I changed `width`):

```jsx
const SearchField = styled.div`
  // other CSS

  width: ${props => props.active ? '100%' : 'initial'};
`
```

Reply:
Right, that's how it works... This is definitely better!